### PR TITLE
[Enhancement] Modify RPC timeout value and GC expiration time for cloud native table

### PR DESCRIPTION
**Pull request by sduzh**
Prevent GC from removing files generated by in-progress compaction tasks.

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: be/src/storage/lake/gc.cpp [hunk 1]
```diff
@@ -214,7 +214,7 @@ static StatusOr<std::set<std::string>> find_orphan_datafiles(TabletManager* tabl
                                                              const std::vector<std::string>& txn_logs) {
     ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(root_location));
     const auto now = std::time(nullptr);
-    const auto expire_seconds = config::lake_gc_segment_expire_seconds;
+    const auto expire_seconds = std::max<int64_t>(config::lake_gc_segment_expire_seconds, 86400);
```
Comment (wyb) [inline be/src/storage/lake/gc.cpp:217]: better add some comments in config.h

File: fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionScheduler.java [hunk 2]
```diff
@@ -154,9 +154,13 @@ private void schedule() {
                 iterator.remove();
                 context.setFinishTs(System.currentTimeMillis());
                 history.offer(CompactionRecord.build(context));
-                if (LOG.isDebugEnabled()) {
+                long cost = context.getFinishTs() - context.getStartTs();
+                if (cost >= /*60 minutes=*/3600000) {
+                    LOG.info("Removed published compaction. {} cost={}s running={}", context.getDebugString(),
+                            cost / 1000, runningCompactions.size());
+                } else if (LOG.isDebugEnabled()) {
                     LOG.debug("Removed published compaction. {} cost={}ms running={}", context.getDebugString(),
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionScheduler.java:162]: `cost={}s`, same as the info log.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 6 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|0|6|00.00%|[157, 158, 159, 160, 161, 163]|

File: fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionScheduler.java (hunk 2 above)
Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionScheduler.java]: updated

File: be/src/storage/lake/gc.cpp (hunk 1 above)
Comment (sduzh) [reply be/src/storage/lake/gc.cpp:217]: updated

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 7 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|0|7|00.00%|[157, 158, 159, 160, 161, 162, 163]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 7 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|0|7|00.00%|[157, 158, 159, 160, 161, 162, 163]|

**Review by wyb — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**